tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+skipsdist = True
+envlist = lint,py3
+
+[tox:travis]
+3.5: lint,py3
```

Comment:
may need to include 3.6 and 3.7 here as well as in addition to .travis.yml

Reply:
done